### ISSUE TITLE
user_ldap WebUIAdminSharingSettingsContext missing in acceptance tests

### ISSUE THREAD

**Issue by phil-davis** [owncloud/user_ldap#438]
CI failed last night with: https://drone.owncloud.com/owncloud/user_ldap/1700/929
```
--- FeatureContext has missing steps. Define them with these snippets:

    /**
     * @Given the administrator has browsed to the admin sharing settings page
     */
    public function theAdministratorHasBrowsedToTheAdminSharingSettingsPage()
    {
        throw new PendingException();
    }

    /**
     * @When the administrator adds :arg1 as a trusted server using the webUI
     */
    public function theAdministratorAddsAsATrustedServerUsingTheWebui($arg1)
    {
        throw new PendingException();
    }

    /**
     * @When the administrator deletes url :arg1 from the trusted server list using the webUI
     */
    public function theAdministratorDeletesUrlFromTheTrustedServerListUsingTheWebui($arg1)
    {
        throw new PendingException();
    }

    /**
     * @Then a trusted server error message should be displayed on the webUI with the text :arg1
     */
    public function aTrustedServerErrorMessageShouldBeDisplayedOnTheWebuiWithTheText($arg1)
    {
        throw new PendingException();
    }
```

**Comment by phil-davis** [comment]
Those steps were added in core by https://github.com/owncloud/core/pull/35826

**Comment by phil-davis** [comment]
Fixed by #439
